pkg/providers/apisix/translation/apisix_route.go

```
@@ -136,8 +137,10 @@ func (t *translator) translateHTTPRouteV2beta2(ctx *translation.TranslateContext
 			if !plugin.Enable {
 				continue
 			}
-			if plugin.Config != nil {
-				pluginMap[plugin.Name] = plugin.Config
+			if plugin.Config.Size() > 0 {
+				config := map[string]interface{}{}
+				_ = json.Unmarshal(plugin.Config.Raw, &config)
```

Why the error is ignored?